default-go-lint.mk

```diff
@@ -14,14 +14,14 @@
 
 
 GOLANG_CI_VER ?= v1.52
-GIT_ROOT_DIR := $(realpath $(dir $(lastword $(MAKEFILE_LIST))))
-include $(GIT_ROOT_DIR)/detect-container-runtime.mk 
+GIT_ROOT_DIR ?= $(dir $(lastword $(MAKEFILE_LIST)))
+include $(GIT_ROOT_DIR)/detect-container-runtime.mk
 
 # Install link at https://golangci-lint.run/usage/install/ if not running inside a container
 .PHONY: lint
 lint: ## Run Go linter against the codebase
 ifeq ($(CONTAINER_RUNNABLE), 0)
-	$(CONTAINER_RUNTIME) run -it -v "$(GIT_ROOT_DIR):$(GIT_ROOT_DIR)" -w "$(CURDIR)" docker.io/golangci/golangci-lint:${GOLANG_CI_VER}-alpine \
+	$(CONTAINER_RUNTIME) run -it -v "$(abspath $(GIT_ROOT_DIR)):$(abspath $(GIT_ROOT_DIR))" -w "$(CURDIR)" docker.io/golangci/golangci-lint:${GOLANG_CI_VER}-alpine \
```

Comment:
FYI I have podman installed on my Fedora and SELinux is enabled as well. The one way that I was able to run either make lint or make unit was by using the option --security-opt label=disable for the run command. This option is supported for both docker and podman. It would be great if we could add this option. THanks